### PR TITLE
Merge master into refactor

### PR DESCRIPTION
**Pull request by Stanzilla**
Figured why not, merry christmas or so :)

### CODE REVIEW

**Comment by squidfunk** [comment]
Thanks! However, your fixed go beyond simply merging master back in, care to elaborate?

**Comment by Stanzilla** [comment]
The build would not complete without that makefile change and the icon location I need to look into, I did not change that.

**Comment by Stanzilla** [comment]
Fixed, build just needed a clean before running it.

File: material/base.html [hunk 1]
```diff
@@ -193,7 +193,7 @@ <h2 id="__source">{{ lang.t("meta.source") }}</h2>
               {% if language == "ja" %}
                 <script src="{{ (path ~ 'tinyseg.js') | url }}"></script>
               {% endif %}
-              {% if language in ("da","de","du","es","fi","fr","hu","it","ja","jp","nl","no","pt","ro","ru","sv","tr") %}
+              {% if language in ("hu","ru","es","ja","ro","it","jp","no","de","sv","nl","vi","ar","tr","da","fi","du","pt","fr") %}
```
Comment (squidfunk) [inline material/base.html]: Uh, support for Arabic. Nice.

**Pull request merged**